packages/listbox/__tests__/listbox.test.tsx

```diff
@@ -197,6 +197,10 @@ describe("<Listbox />", () => {
           </Listbox>
         );
         expect(container.querySelector("input")).toBeTruthy();
+        expect(container.querySelector("input")).toHaveAttribute(
```

Comment:
Can we instead test that the input is visibly hidden with `expect(container.querySelector("input")).not.toBeVisible()`? I'll probably change the implementation of this in the near future to address arrow form navigation on iOS.